src/main/java/de/paginagmbh/epubchecker/PaginaEPUBChecker.java

```diff
@@ -19,15 +19,15 @@
  *
  * @author      Tobias Fischer
  * @copyright   pagina GmbH, Tübingen
- * @version     2.0.3
- * @date        2021-01-07
+ * @version     2.0.4
+ * @date        2021-01-31
  */
 public class PaginaEPUBChecker {
 
 	// +++++++++++++++++++++++++ DON'T FORGET TO UPDATE EVERYTIME ++++++++++++++++++ //	
 
-	public static final String PROGRAMVERSION = "2.0.3";
-	public static final String VERSIONDATE = "07.01.2021";
+	public static final String PROGRAMVERSION = "2.0.4";
+	public static final String VERSIONDATE = "31.01.2021";
 	public static final String PROGRAMRELEASE = "Beta";	// "" or "Beta"
```

Comment:
Blocker! must be changed from `"Beta"` to `""` BEFORE merging this PR!

Reply:
Thanks, that should of course have been updated.

Honestly though, what is that "Beta" option even still there for? The server side update download links for both versions have been pointing to the same file for quite a while now.

Reply:
I think the main feature here is to display "Version x.x.x Beta" in the About box when I built pre-release test packages for users who wrote emails, reported bugs, requested features, so that they can test on a different "release channel". This is clearly taken from early parsX ages, where we used this heavily...
However, I never really "released" beta updates publicly.

Reply:
I see. Thank you for the explanation!